.github/workflows/lint-react.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: "Lint react"
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - 'docs/*'
+  push:
+    branches:
+      - 'main'
+      - 'releases/**'
+
+jobs:
+  lint-react:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
+      # Explicitly caching node_modules manually as actions/setup-node does not
+      - name: Cache node_modules
+        uses: actions/cache@v3
+        with:
+          key: webui/react/node_modules.${{ hashFiles('webui/react/package-lock.json') }}
+          path: webui/react/node_modules
+      - name: Install dependencies
+        run: test -d webui/react/node_modules || make -C webui/react get-deps-npm
```

Comment:
Why `test -d` rather than skipping the step with `if: steps.cache.outputs.cache-hit != 'true'` (and adding an id to the cache step) to use the cache hit output?

Reply:
No particular reason -- hadn't thought to double check if the cache action would report a hit or not, I'll switch it over before merging.